OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -0,0 +1,170 @@
+SOA on Docker
+=============
+
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.3.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle Fusion Middleware Infrastructure 12.2.1.3.0.
+
+At the end of this configuration there may be 3 containers running: 
+1. Oracle Database Container (Only when container based Database is used)
```

Comment:
(Optional: only when RCU schema is created in a database running in a container)

I would add optional just to make it clear